### DISCUSSION TITLE
Domain linking issue

### DISCUSSION THREAD

**Discussion by scriptyyy13** [deta/cloud-docs#417] [Help] [answered]
About 32 hours ago I set up a record on my domain. `A at 75.2.69.226`. But for some reason it did not get attached, although I indicated it in the micro settings. I used Cloudflare and DNS only mode

**Comment by AyushSehrawat** [comment, marked as answer]
Delete that and recreate again, micro will probably give new ip address, put it as A record (dns only) in cloudfare.

Like 'abc.abc.abc' is a custom domain for your micro, micro gives ip, put it as A record with dns only.

It should happen fast, but if it didn't maybe wait for few hours.

**Reply by scriptyyy13** [reply]
I saw in the documentation that some domain zones are not supported. I would like to know if `.su` is supported? Or do you not have such information?
